reo/validators.py

```diff
@@ -366,6 +367,81 @@ class ValidateNestedInput:
     #         }
     #     }
 
+    fuel_conversion_per_mmbtu = {
+                "natural_gas":116.9,
+                "landfill_bio_gas":114.8,
+                "propane":138.6,
+                "diesel_oil": 163.1
+            }
+
+    fuel_conversion_per_gal = {
+                'diesel_oil':22.51
+            }
+
+    prime_mover_defaults = {'recip_engine': {
```

Comment:
can these defaults be defined in nested_inputs.py with the rest of the default values? (It would be nice to have all defaults in one place).

Reply:
@NLaws I'm moving these outside of validators.py in the chp_size_class_cp branch but not in nested_inputs. CHP defaults are a bit more involved with the prime_mover (4x) AND size_class dependence (3-4x), and I didn't want to clutter nested_inputs. We can discuss if it should go there.

Reply:
@Bill-Becker Are these defaults for non-_user_ inputs? Then they do not need to be in nested_inputs.

Reply:
@NLaws They are for user inputs.